### DISCUSSION TITLE
What do any of these "feed" events have to do with me?

### DISCUSSION THREAD

**Discussion by TomArrow** [community/community#54114] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In the past the homepage showed me if someone forked one of my repos or followed me or something. Was nice to notice that people like my projects every now and then, however rarely it occurred. 

Now it shows some random stuff about a random  person with whom I might have interacted once 5 years ago following another person I never heard of. What? Why would I care? What does that have to do with me? A full feed plastered with information that is utterly irrelevant and uninteresting to me and anything that might be interesting to me buried under thousands of messages so that I'll never see it. What's the logic of this?

**Comment by meppu** [comment]
You can try filtering feeds for your personal preference. Go to https://github.com/ and at the right side of the bar, click "Filter" dropdown. Then close whatever you don't want to see and click "Apply changes"

![image](https://user-images.githubusercontent.com/112294217/235380158-de4458c8-1d91-40bb-a5b0-b2183f56cac9.png)
![image](https://user-images.githubusercontent.com/112294217/235380211-030495b3-726c-4486-9638-e51a1784be05.png)
